SPRINTUSER_README.md

```diff
@@ -1,32 +1,94 @@
-* Windows 10 Home (and other versions like Server): Please use Docker Toolbox (bundled). You must start up Docker Toolbox using the "Docker Toolbox Quickstart"
-* [Git for Windows](https://gitforwindows.org/): git-bash is used for all installation scripts, and of course you'll need git to be able to develop patches. [Helpful git setup info at github](https://help.github.com/articles/set-up-git/). 
-* ddev Windows Installer (bundled)
-
-**macOS and Linux Prerequisites**
-* Docker-for-Mac (bundled) or for Linux. See [detailed instructions](https://ddev.readthedocs.io/en/latest/users/docker_installation/)
-
-**Installation and Startup:**
-
-1. Install Docker CE if you don't already have it
-  - **macOS/Windows**: There may be installers supplied with this package
-  - **Linux**: See [Linux instructions](https://docs.docker.com/install/#docker-ce)
-2. Start Docker (it must be running to install ddev)
-3. Install DDEV-Local:
-  - Run install_ddev.sh via terminal (git-bash on Windows) from the directory where this SPRINTUSER_README.md is with `./install_ddev.sh` - On Windows also run the ddev_windows_installer bundled.
-4. Create Drupal 8 instance to use during sprint:
-  - cd to `~/sprint` and run `./start_sprint.sh`
-5. There should be a new sprint folder named `sprint-YYYYMMDD-HHMM` (with today's date and the time you ran `start_sprint`). Start the Drupal environment by using cd to move to the dated sprint folder, and run `./start_clean.sh`
-6. After start_clean is finished, it will provide you with a set of URLs and further instructions for using your sprint environment.
+## Prerequisites
+
+* A computer with 8gb memory or greater.
+* Windows 7 or higher, MacOS Sierra or higher or a recent/stable Linux distribution.
+* A robust code editor such as Visual Studio Code, Atom, PhpStorm, Netbeans, vim or emacs (this may be provided as part of this package).
+
+**⚠️ You can still install and contribute to Drupal even without these requirements using [Drupal's quick-start command](https://www.drupal.org/docs/8/install/quick-start-launch-a-local-demo-version-of-drupal-8-using-4-brief-steps)**!
+
+## Getting Started
+
+1. [Get The Files](#get-the-files)
+2. [Extract drupal_sprint_package](#extract-files)
+3. [Install Docker and Other Requirements](#install)
+4. [Open Terminal](#open-terminal)
+5. [Install Contribution Tools](#install-tools)
+
+<a name="get-the-files"></a>
+### 1. Get The Files
+
+* USB Drive - Copy drupal_sprint_package-RELEASE.zip and Docker installer for your Operating System from the USB drive to your Desktop.
+* ResilioSync - Find the folder/directory that was downloaded and copy its contents to your Desktop.
+* GitHub - Download drupal_sprint_package-RELEASE.zip from https://github.com/drud/quicksprint/releases.
+
+<a name="extract-files"></a>
+### 2. Extract drupal_sprint_package directory
+
+1. Extract the drupal_sprint_package-RELEASE.zip file to your Desktop. It is important that these files exist within your "user directory".
```

Comment:
Why is that important? I usually do it in /tmp...

Reply:
Whoops, I carried this over from old documentation in docker-core-sprint where I was running docker directly from the package directory (so it needed to be in the user directory for Windows and MacOS). I re-worded.